packages/mobile/e2e/src/usecases/WalletConnectV1.js

```diff
@@ -323,6 +323,7 @@ export default WalletConnect = () => {
     await element(by.text('Connected Apps')).tap()
     await element(by.text('Tap to Disconnect')).tap()
     await element(by.text('Disconnect')).tap()
+    await element(by.id('BackChevron')).tap()
```

Comment:
after apps are disconnected, the user is still on the Sessions screen but the next assertion for `ConnectedApplications/value` is on the Settings screen. because this assertion is in the `afterAll` hook, the tests failed without being captured by the test reports or test artifacts. i think for correctness the command to navigate back to the settings screen should exist...

(i also ran this test locally on the main branch, and it seems that the tests pass even though the app is still on the Sessions screen and the `ConnectedApplications/value` is not found :/ i'm not sure why it can pass, but the behaviour of disconnecting a connected app has not changed in this PR)